### DISCUSSION TITLE
Enhancement: Option to allow left sidebar to be resized.

### DISCUSSION THREAD

**Discussion by netnem** [danny-avila/LibreChat#3038] [Feature Requests & Suggestions]
### What features would you like to see added?

I think the request is a simple one, but it drives me absolutely bonkers that I have to "hover over" every left side entry in order to read the full summary of the chat.

Eg:

![image](https://github.com/danny-avila/LibreChat/assets/32517635/2a1df011-9979-4ee0-8383-00ba172ae8f4)

I am unable to read the full summary without hovering over the item.  I realize that chatGPT is the exact same way, so I think this should still be the "default" option.  But having an option to "Allow resizeable Left bar" that behaves like the right sidebar. would be desirable.  

### More details

Add an option in menu to "allow resizable left bar" to mimick how right side bar is resizable.  This will allow being able to read through hundreds of chat summaries much faster.

### Which components are impacted by your request?

UI

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
Thanks I'll look into. Sadly it's not as simple as it seems to allow resizing, and it may compromise the "smooth" animation that it currently has when minimizing (which is why the right-hand side nav doesn't have it).

**Comment by netnem** [comment]
Thanks @danny-avila,

What do you think about setting the pixels as user defined variables? 

NAV_BAR_WIDTH=400px or w/e

Alternatively this could be a UI option instead of a .env setting.  

I tested out updating the divs/css directly in my local branch and seems to work well for my purposes.

![image](https://github.com/danny-avila/LibreChat/assets/32517635/2e8cc4c6-789c-4c7c-af58-2dba8bbb71fe)

This way It wouldn't have to be "resizable" like the right side bar, just adjustable.

**Comment by netnem** [comment]
For anyone else looking to do this quick and dirty:

```
docker exec LibreChat /bin/sh -c '/usr/bin/find /app/client/dist/ -iname "index*js" -print0 | xargs -0 sed -i "s/260px/400px/g"' 
docker exec LibreChat /bin/sh -c '/usr/bin/find /app/client/dist/ -iname "index*css" -print0 | xargs -0 sed -i "s/260px/400px/g"'
```
